### ISSUE TITLE
[Bug] TypeORM hooks not being triggered with nestjs-query

### ISSUE THREAD

**Issue by GustavoLofrano** [doug-martin/nestjs-query#380]
![image](https://user-images.githubusercontent.com/36713355/87597052-7f9cde80-c6c7-11ea-8148-7dd9f6c32d3c.png)

In UserEntity:

![image](https://user-images.githubusercontent.com/36713355/87596951-55e3b780-c6c7-11ea-9af2-9c374167d91d.png)

Don't hash the password in table:

![image](https://user-images.githubusercontent.com/36713355/87597006-6c8a0e80-c6c7-11ea-8655-85366cb3124b.png)

without nestjs-query they work:
![image](https://user-images.githubusercontent.com/36713355/87597168-b541c780-c6c7-11ea-8ef1-dc4c127e4599.png)

**Comment by doug-martin** [comment]
@GustavoLofrano I just tested this locally and it worked fine. If you can provide a repo I can try and help to diagnose the issue.

**Comment by GustavoLofrano** [comment]
I created the repository

https://github.com/GustavoLofrano/school-app.graphql.git

**Comment by GustavoLofrano** [comment]
I think I found the problem.
is when using a class UserService, if commenting the code, will work.
I'm creating the Service class, in case I need it in the future, I'm standardizing.


it works
```
@Module({
  imports: [
    NestjsQueryGraphQLModule.forFeature({
      imports: [NestjsQueryTypeOrmModule.forFeature([UserEntity])],
      //services: [UserService],
      resolvers: [
        {
          EntityClass: UserEntity,
          //ServiceClass: UserService,
          DTOClass: UserDTO,
          CreateDTOClass: UserCreateDTO,
          UpdateDTOClass: UserUpdateDTO,
          pagingStrategy: PagingStrategies.CURSOR,
        },
      ],
    }),
  ],
})
export class UserModule {}
```


does not work
```
@Module({
  imports: [
    NestjsQueryGraphQLModule.forFeature({
      imports: [NestjsQueryTypeOrmModule.forFeature([UserEntity])],
      services: [UserService],
      resolvers: [
        {
          EntityClass: UserEntity,
          ServiceClass: UserService,
          DTOClass: UserDTO,
          CreateDTOClass: UserCreateDTO,
          UpdateDTOClass: UserUpdateDTO,
          pagingStrategy: PagingStrategies.CURSOR,
        },
      ],
    }),
  ],
})
export class UserModule {}
```

**Comment by doug-martin** [comment]
Thanks! I'll take a look later today.

**Comment by doug-martin** [comment]
@GustavoLofrano thanks again for the bug report and repo. I was able to reproduce and fix! If you update to version `v0.17.2` it should work.

Let me know if you run into any additional issues.

**Comment by GustavoLofrano** [comment]
@doug-martin. I'll use your package a lot. I'm starting a big project and the graphql base will be your package. The productivity gain that I will have is very good. Whenever I find something, I will try to help.
